proto/action.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package action;
+
+service action {
```

Comment:
Here you want to define a service for scheduling an action:

- It is a scheduler service and should be defined as part of the scheduler API
- Its name should describe the service that it exposes, e.g. `SchedulerService`